config/celery_settings.py

```diff
@@ -1,23 +1,35 @@
 from __future__ import absolute_import
+
 import os
-# import ssl
+import ssl
+
+import environ
 from celery import Celery
 from django.conf import settings
 
+env = environ.Env(
+    # set casting, default value
+    DEBUG=(bool, False),
+    USE_S3=(bool, False),
+    RENDER=(bool, False),
+)
+
+environ.Env.read_env()
+
+your_env = env('ENVIRONMENT')
+
 os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'config.settings')
-# for production
-# app = Celery(
-#     "config",
-#     broker_use_ssl={
-#         'ssl_cert_reqs': ssl.CERT_NONE
-#     },
-#     redis_backend_use_ssl={
-#         'ssl_cert_reqs': ssl.CERT_NONE
-#     }
-# )
-
-# for development
-app = Celery('config')
+
+if your_env == 'prod':
+    app = Celery(
+        'config',
+        broker_use_ssl={'ssl_cert_reqs': ssl.CERT_NONE},
+        redis_backend_use_ssl={'ssl_cert_reqs': ssl.CERT_NONE},
+    )
+elif your_env == 'dev':
+    app = Celery('config')
+else:
+    raise ValueError('ENVIRONMENT in .env file should be either "dev" or "prod".')
```

Comment:
@GitLukeW Would we need to have a staging env set up for the staging site or would we just use prod as well for it?

Reply:
I would assume the staging environment would use the same app as prod, but that's just a guess.

Reply:
I guess my question here is why are we changing this file?

Reply:
I'm changing it because we want to remove all commented out code. The production code was important, so I created a new ENVIRONMENT env variable that we could check to see if it should be run. @esparr does make a good point, though. Is there anything that needs to run differently for staging?

Reply:
@GitLukeW Do you see any issues with this variable getting added to production? Seems like Staging would be set up like development for this one since we do not have a redis/celery setup on staging. 

One thing we will need to look out for is testing that celery is working once we get it on production (staging does not have celery/redis setup) I wonder if adding an integration test would be helpful to make sure this is working moving forward?

Reply:
Sorry for the confusion. This can actually be taken out and just use the:

app = Celery('config')